site/zenodo_rdm/support/utils.py

```diff
@@ -0,0 +1,15 @@
+from flask import current_app
+
+
+def render_template_to_string(template, context):
+    """Render a Jinja template with the given context."""
+    template = current_app.jinja_env.get_or_select_template(template)
+    return template.render(context)
+
+
+def format_user_email(email, name):
+    """Format the user's email as 'Full Name <email>' or 'email'."""
+    if name:
+        email = "{name} <{email}>".format(name=name, email=email)
+    # Remove commas (',') since they mess with the "TO" email field
+    return email.replace(",", "")
```

Comment:
what if there are several email addresses passed in the email field? can this happen?

Reply:
I don't think so, given that the form is currently validated by marshmallow. Adding commas yields an error in marshmallow so I will remove that `replace`.